modules/widgets/eu-cookie-law.php

```diff
@@ -0,0 +1,162 @@
+
+		require( dirname( __FILE__ ) . '/eu-cookie-law/form.php' );
+	}
+
+	public function update( $new_instance, $old_instance ) {
+		$instance = array();
+
+		if ( in_array( $new_instance['hide'], array( 'button', 'scroll', 'time' ) ) ) {
+			$instance['hide'] = $new_instance['hide'];
+		}
+
+		if ( isset( $new_instance['hide-timeout'] ) ) {
+			// time can be a value between 5 and 1000 seconds
+			$instance['hide-timeout'] = min( 1000, max( 3, intval( $new_instance['hide-timeout'] ) ) );
+		}
+
+		if ( in_array( $new_instance['text'], array( 'default', 'custom' ) ) ) {
+			$instance['text'] = $new_instance['text'];
+		}
+
+		if ( isset( $new_instance['customtext'] ) ) {
+			$instance['customtext'] = mb_substr( $new_instance['customtext'], 0, 4096 );
+		} else {
+			$instance['text'] = 'default';
+		}
+
+		if ( isset( $new_instance['color-scheme'] ) ) {
+			$instance['color-scheme'] = 'negative' === $new_instance['color-scheme'] ? 'negative' : 'default';
+		} else {
+			$instance['color-scheme'] = 'default';
+		}
+
+		if ( in_array( $new_instance['policy-url'], array( 'default', 'custom' ) ) ) {
+			$instance['policy-url'] = $new_instance['policy-url'];
+		}
+
+		if ( isset( $new_instance['custom-policy-url'] ) ) {
+			$instance['custom-policy-url'] = esc_url( $new_instance, array( 'http', 'https' ) );
+
+			if ( strlen( $instance['custom-policy-url'] ) < 10 ) {
+				unset( $instance['custom-policy-url'] );
+				$instance['policy-url'] = 'default';
+			}
+		} else {
+			$instance['policy-url'] = 'default';
+		}
+
+		if ( isset( $new_instance['policy-link-text'] ) ) {
+			$instance['policy-link-text'] = trim( mb_substr( $new_instance['policy-link-text'], 0, 100 ) );
+		}
+
+		if ( empty( $instance['policy-link-text'] ) || $instance['policy-link-text'] == $this->defaults['policy-link-text'] ) {
+			unset( $instance['policy-link-text'] );
+		}
+
+		if ( isset( $new_instance['button'] ) ) {
+			$instance['button'] = trim( mb_substr( $new_instance['button'], 0, 100 ) );
+		}
+
+		if ( empty( $instance['button'] ) || $instance['button'] == $this->defaults['button'] ) {
+			unset( $instance['button'] );
+		}
+
+		// show the banner again if a setting has been changed
+		setcookie( self::$cookie_name, '', time() - 86400, '/' );
+
+		return $instance;
+	}
+
+	public static function add_consent_cookie() {
+		if ( ! isset( $_POST['eucookielaw'] ) || 'accept' !== $_POST['eucookielaw'] ) {
+			return;
+		}
+
+		if ( ! isset( $_POST['_wpnonce'] ) || ! wp_verify_nonce( $_POST['_wpnonce'], 'eucookielaw' ) ) {
+			return;
+		}
+
+		// Cookie is valid for 30 days, so the user will be shown the banner again after 30 days
+		setcookie( self::$cookie_name, current_time( 'timestamp' ), time() + self::$cookie_validity, '/' );
+
+		wp_safe_redirect( $_POST['redirect_url'] );
+	}
+
+	function enqueue_style() {
+		wp_enqueue_style( 'eu-cookie-law-style', plugins_url( 'eu-cookie-law/style.css', __FILE__ ), array(), '20170403' );
```

Comment:
Jetpack automatically concatenates all CSS files into a single `jetpack.css` file. To add your CSS to the concatenated file, you'll need to add it [here](https://github.com/Automattic/jetpack/blob/master/gulpfile.js#L291) and [here](https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L58).